ckcp/gitops.sh

```diff
@@ -279,8 +287,8 @@ install_triggers_controller(){
   #############################################################################
   # Create kcp-kubeconfig secret for triggers controller
   #############################################################################
-  oc create namespace triggers -o yaml --dry-run=client | oc apply -f- &>/dev/null
-  oc create secret generic ckcp-kubeconfig -n triggers --from-file "$KUBECONFIG_KCP" --dry-run=client -o yaml | oc apply -f - &>/dev/null
+  oc create namespace triggers -o yaml --dry-run=client | oc apply -f - --wait &>/dev/null
+  oc create secret generic ckcp-kubeconfig -n triggers --from-file "$KUBECONFIG_KCP" --dry-run=client -o yaml | oc apply -f - --wait &>/dev/null
```

Comment:
what exactly does "--wait" do in lines 280, 281, 290, and 291? (lines creating secrets and namespace)